README.md

```diff
@@ -1,2 +1,26 @@
-# opg-dockerise-go
-OPG Dockerise Go repo: Managed by opg-org-infra &amp; Terraform
+# Background
+
+Our production environments utilise AWS cloud platform with the infrastructure being created and controlled by Hashicorp's Terraform. Our deployment process takes the app code repository, generates a series of docker images which are then placed into AWS Elastic Cluster Service.
+
+# Task
+
+This repository contains a Go application (with `./app` folder) that acts as an API; it is configured to respond on port `8080` and has two endpoints (`/` and `/status`) that return `json` content.
+
+The `/status` endpoint returns different content and `http` response code depending on the value of an environment variable called `APP_STATUS`. When `APP_STATUS=OK` the `/status` endpoint will return a `200` code and message, otherwise if will return a `500` code.
```

Comment:
```suggestion
The `/status` endpoint returns different content and `http` response code depending on the value of an environment variable called `APP_STATUS`. When `APP_STATUS=OK` the `/status` endpoint will return a `200` code and message, otherwise it will return a `500` code.
```